client/components/social-buttons/google.js

```diff
@@ -106,12 +113,23 @@ class GoogleLoginButton extends Component {
 						this.props.responseHandler( currentUser, false );
 					}
 
+					if ( this.state.isClicked ) {
+						// Make sure that handleClick Call happens on the next tick so that the popup always launches.
+						setTimeout( () => {
+							this.handleClick( this.state.isClicked );
+						}, 10 );
```

Comment:
If you just want to wait for the next event loop tick, you can omit the delay parameter altogether.